git/repo.go

```diff
@@ -86,7 +86,7 @@ func (r Repo) Clone(c Config) (*Checkout, error) {
 	}
 
 	notesRef, err := getNotesRef(repoDir, c.NotesRef)
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "Couldn't find remote ref") {
```

Comment:
Yeah, but that git command doesn't do any remote operations, it just tells you what the full ref would be for the shortcut name you give it, and it doesn't even have to exist locally. So I can do, for example,

```
$ git notes --ref=madeupjustnow get-ref
refs/notes/madeupjustnow
```